### PR TITLE
Lwdev 10574 upgrade requests

### PR DESCRIPTION
**Pull request by samodovdi**

### CODE REVIEW

File: src/Lykke.Service.Tier/Workflow/Projections/ClientDepositsProjection.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                await _clientAccountClient.ClientSettings.SetCashOutBlockAsync(new CashOutBlockRequest
+                {
+                    ClientId = evt.ClientId, CashOutBlocked = false, TradesBlocked = true
+                });
+            }
+        }
+
+        private async Task<double> GetClientLimitSettingsAsync(string clientId, AccountTier tier, string country)
+        {
+            throw new System.NotImplementedException();
```
Comment (polonzer) [inline src/Lykke.Service.Tier/Workflow/Projections/ClientDepositsProjection.cs:64]: Why not implemented?

File: src/Lykke.Service.Tier/Workflow/Projections/TierUpgradeRequestProjection.cs [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+                                "EN", new { FullName = personalData.FullName, DocumentsAsHtml = documentsAsHtml, Year = DateTime.UtcNow.Year });
+                        }
+
+                        if (pushEnabled)
+                            pushTemplateTask = _templateFormatter.FormatAsync("PushKycNeedDocumentsTemplate", clientAcc.PartnerId, "EN", new { });
+
+                        type = NotificationType.KycNeedToFillDocuments.ToString();
+                        break;
+                    case KycStatus.Rejected:
+//                        emailTemplateTask = _templateFormatter.FormatAsync("TierUpgradedTemplate", clientAcc.PartnerId,
```
Comment (polonzer) [inline src/Lykke.Service.Tier/Workflow/Projections/TierUpgradeRequestProjection.cs:109]: Why commented?

File: src/Lykke.Service.Tier/Workflow/Projections/ClientDepositsProjection.cs (hunk 1 above)
Comment (samodovdi) [reply src/Lykke.Service.Tier/Workflow/Projections/ClientDepositsProjection.cs:64]: implemented in the other task

File: src/Lykke.Service.Tier/Workflow/Projections/TierUpgradeRequestProjection.cs (hunk 2 above)
Comment (samodovdi) [reply src/Lykke.Service.Tier/Workflow/Projections/TierUpgradeRequestProjection.cs:109]: implemented in the other task

**Review by polonzer — APPROVED** [review]

**Pull request merged**